src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Excavator is doing its work.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Excavator has stopped working.");

```

Comment:
The stop message should also be more informative. Consider something like: 'Excavator has been turned off and can no longer dig.'